src/components/LibraryDetailPage.tsx

```diff
@@ -43,7 +43,8 @@ export class LibraryDetailPage extends React.Component<LibraryDetailPageProps, L
 
   renderInfo() {
     let library = this.props.fullLibrary || this.props.library;
-    let fields = Object.keys(library).map(label =>
+    // Only create LibraryDetailItems for fields which actually have a value.
```

Comment:
When is the case when there's a library without a label value?

Reply:
Optional fields.  For example: a library can, in theory, have a description...but if this library doesn't happen to have a description, then there's no point making a `LibraryDetailItem` that just says "description" with no value.

Reply:
Ah you know what mixed me up? That you are using the `library` name in `label => library[label]`. If it was `field[label]` it would be easier to know what is going on.

Reply:
But It makes sense.